ignite/cmd/chain_serve.go

```diff
@@ -78,11 +83,19 @@ production, you may want to run "appd start" manually.
 	c.Flags().Bool(flagGenerateClients, false, "generate code for the configured clients on reset or source code change")
 	c.Flags().Bool(flagQuitOnFail, false, "quit program if the app fails to start")
 	c.Flags().StringSlice(flagBuildTags, []string{}, "parameters to build the chain binary")
+	c.Flags().BoolP(flagDaemon, "d", false, "prepare chain serve to run in background (no UI, no stdin, listens for SIGTERM, implies --yes)")
+	c.Flags().StringP(flagOutputFile, "o", "", "output file for the daemon logs (default: stdout)")
```

Comment:
Why not use only the output flag? Since the `deanom` flag is the same as the `yes` flag with `output`.

Reply:
Because it could be confusing that having an output doesn't use the hi. Unless you think this is understandable enough.

Reply:
implemented

Reply:
IMHO, only a `log-output` flag is sufficient

Reply:
or maybe `log-output-file`